pkg/controller/risingwave_controller.go

```diff
@@ -408,6 +360,11 @@ func (c *RisingWaveController) reactiveWorkflow(risingwaveManger *object.RisingW
 }
 
 func (c *RisingWaveController) SetupWithManager(mgr ctrl.Manager) error {
+	gvk, err := apiutil.GVKForObject(&risingwavev1alpha1.RisingWave{}, c.Client.Scheme())
+	if err != nil {
+		return fmt.Errorf("unable to find gvk for RisingWave: %w", err)
```

Comment:
`%w` unwraps the error, right?

Reply:
No, "%w" wraps the error, in this example it would return an error that wraps the original `err` and has a message of `"unable to find gvk for RisingWave: " + err.Error()`.